src/library/scala/collection/SeqView.scala

```diff
@@ -30,8 +30,6 @@ trait SeqView[+A] extends SeqOps[A, View, View[A]] with View[A] {
   def appendedAll[B >: A](suffix: SeqView.SomeSeqOps[B]): SeqView[B] = new SeqView.Concat(this, suffix)
   def prependedAll[B >: A](prefix: SeqView.SomeSeqOps[B]): SeqView[B] = new SeqView.Concat(prefix, this)
 
-  override def sorted[B >: A](implicit ord: Ordering[B]): SeqView[A] = new SeqView.Sorted(this, ord)
```

Comment:
I don't think you can get rid of this method - it will return a `View[A]` rather than a `SeqView[A]`, which is undesirable.

Reply:
Yeah I was realizing this after the fact. It's also annoying because you have to actually override `sortedBy` and `sortedWith` because they return `View[A]` as well